main.py

```diff
@@ -20,12 +20,6 @@ def main(config):
     random.seed(None)
     # Set other random seeds
     set_seeds(config.seed)
-    # Log config
-    # TODO: Move log config to Logger
-    log_config = flatten_config(OmegaConf.to_container(config, resolve=True), sep="/")
-    log_config = {"/".join(("config", key)): val for key, val in log_config.items()}
```

Comment:
Are you planning to revert / use `flatten_config` in the future? If not how about deleting it to clean things up?